modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -20,6 +20,8 @@ you add a new version to an existing asset.
 
 For an HTTP API, because the asset is only an endpoint, it makes no sense to increase the patch or minor versions in every new publication. HTTP APIs therefore only have major version increases and also changes to the API version.
 
+When creating a new version of an asset, the name, icon, and description of the previous version are used. To change these properties use the instructions in xref:to-describe-an-asset.adoc[Describe an Asset]. Any changes to the asset name, icon, or description apply to all versions of the asset.
```

Comment:
"When creating a new version of an asset, the name, icon, and description of the previous version are used. To change these properties use the instructions in xref:to-describe-an-asset.adoc[Describe an Asset]. Any changes to the name, icon, or description apply to all versions of the asset."

I'm trying to avoid calling it "asset" name, since technically, it is the Name of the asset portal. 

@mcampo , @LucasLudueno : Here're my thoughts. I think we should be very clear about what happens when & why. 

Publish from API Designer:
The first time it's published from API Designer, API Designer reads & uses the name of the asset from the RAML spec, before publishing it to Exchange. The user can edit the name before publishing. But once it's published in Exchange, the name belongs to Exchange, and it's treated as the Asset Portal name. From then on any publication from API Designer to update the version of an asset already in Exchange, will always have the name field disabled, and will reflect the same name from Exchange. The only place the user can edit the name and/or update the icon or description of an asset, will have to be done from the Exchange UI.

Publishing from Exchange UI:
Users can pick a name when they FIRST publish an asset to Exchange from the Exchange UI. From then on, any new addition of a version from the Exchange UI, will have the name disabled. Again, the only place the name, description & icon are updatable is from the exchange Asset details page.

Somehow, somewhere the above needs to be clearly articulated.

Reply:
Good point! Removed the "asset name" from to-change-raml-version.adoc.

I think the best place to discuss it is to-create-an-asset.adoc, so I've added this info there.

Reply:
Agree with Will in discussing it ` to-create-an-asset.adoc`, but, should we explain something related to API Design here in Exchange docs? (just a doubt)

Reply:
In this case the user workflow uses both API Designer and Exchange, so I think it's best to include this much information about it here in the Exchange docs.